From e146e35fccc4e6ec949998698aedeff161875f48 Mon Sep 17 00:00:00 2001
From: Hiroshi Hatake <hatake@clear-code.com>
Date: Tue, 4 Feb 2020 18:38:37 +0900
Subject: [PATCH] Use TLS based SSL context

Signed-off-by: Hiroshi Hatake <hatake@clear-code.com>
---
 src/streams/openssl.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/streams/openssl.c b/src/streams/openssl.c
index 98a3635af..2683f9aa9 100644
--- a/src/streams/openssl.c
+++ b/src/streams/openssl.c
@@ -218,7 +218,7 @@ int git_openssl_stream_global_init(void)
 	 * compatibility. We then disable SSL so we only allow OpenSSL
 	 * to speak TLSv1 to perform the encryption itself.
 	 */
-	if (!(git__ssl_ctx = SSL_CTX_new(SSLv23_method())))
+	if (!(git__ssl_ctx = SSL_CTX_new(TLS_method())))
 		goto error;
 
 	SSL_CTX_set_options(git__ssl_ctx, ssl_opts);
-- 
2.20.1

